docs/guides/publish_service.md

```diff
@@ -0,0 +1,41 @@
+Publishing a service is simply the process of storing the service into a registry (or in other words, a repository), either locally or remotely. The {{open_autonomy}} framework supports three types of registries: local, remote [IPFS](https://ipfs.io/), and remote HTTP. Remote registries allow to make the service available to other developers.
+
+This guide assumes that your terminal is located in the service folder that you want to deploy. that is, the folder containing the `service.yaml` file.
+
+## What will you learn
+In this guide, you will learn how to:
+
+  * Publish a service on a local registry.
+  * Publish a service on an IPFS registry.
+  * Publish a service on an HTTP registry.
+
+## Publish a service locally
+
+!!! info
+    This section will be added soon.
+
+## Publish a service on an IPFS registry
+
+1. **Check the prerequisites.** Ensure that the framework has been initiated with the options `--remote --ipfs`.
+
+2. **Publish the service.** Publish the service by executing the following command within the service parent folder:
+
+    ```bash
+    autonomy publish --remote
```

Comment:
@angrybayblade There is some "homogeneity" mismatch. In init, we need to specify  e.g. --remote --ipfs, whereas in this command, only --remote. Is that the intended ux?

Reply:
I understand, but what if users inits repository with --local, and then he uses  --remote in this command? what repository will be using http or ipfs?